examples/jsm/loaders/GLTFLoader.js

```diff
@@ -1993,12 +1995,94 @@ const ATTRIBUTES = {
 };
 
 const PATH_PROPERTIES = {
+	position: 'position',
 	scale: 'scale',
 	translation: 'position',
 	rotation: 'quaternion',
 	weights: 'morphTargetInfluences'
 };
 
+// TODO move to extension
+const animationPointerDebug = false;
+
+const TARGET_TYPE = {
+	node: 'node',
+	material: 'material',
+	camera: 'camera',
+	light: 'light',
+};
+
+// HACK monkey patching findNode to ensure we can map to other types required by KHR_animation_pointer.
+const find = PropertyBinding.findNode;
```

Comment:
I don't think monkey patching is a good idea. Maybe good to consider either

1. Update the core `findNode`
2. Gave up the perfect extension support and consider partial feature support that the existing `findNode` can handle
3. If we really need monkey patching, it may be good to do it only if `KHR_animation_pointer` extension is used

Reply:
The existing `findNode` can't handle most of what animation_pointer requires unfortunately. I agree that it would be better to update `findNode` as mentioned in my original post, looking forward to ideas which parts should move there. Ideally there's a callback that for example extensions can tap into.

Reply:
I changed the implementation to only monkey patch if the extension is actually used.